### PR TITLE
Add more details to census README

### PR DESCRIPTION
**Pull request by drewsamnick**
These are the things that we discussed in the brain dump meetings that weren't already covered in the README. Let me know if there are other details that you'd like expanded.

### CODE REVIEW

File: dashboard/app/models/census/README.md [hunk 1]
```diff
@@ -10,29 +10,87 @@ The different forms are:
...
 # Summarization
 
 Each school may have multiple data points for the same school year. All of that data is aggregated into a simple summary that says if we believe the school teaches/taught Computer Science in a given year. The possible values are:
 
 * **Yes** - We are fairly confident that the school teaches CS
 * **No** - We are fairly confident that the school **does not** teach CS
 * **Maybe** - The data did not give us a strong indication one way or the other
+* **Historical Yes** - In one of the past two years we were fairly confident that the school taught CS
+* **Historical No** - In one of the past two years we were fairly confident that the school **did not** teach CS
+* **Historical Maybe** - In one of the past two years we were the data did not give us a strong indication one way or the other
```
Comment (breville) [inline dashboard/app/models/census/README.md]: nit: missing words

File: dashboard/app/models/census/README.md [hunk 2]
```diff
@@ -10,29 +10,87 @@ The different forms are:
...
+* [The react component for the  review page](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/census2017/CensusInaccuracyReview.jsx)
+
 # Summarization
 
 Each school may have multiple data points for the same school year. All of that data is aggregated into a simple summary that says if we believe the school teaches/taught Computer Science in a given year. The possible values are:
 
 * **Yes** - We are fairly confident that the school teaches CS
 * **No** - We are fairly confident that the school **does not** teach CS
 * **Maybe** - The data did not give us a strong indication one way or the other
+* **Historical Yes** - In one of the past two years we were fairly confident that the school taught CS
```
Comment (breville) [inline dashboard/app/models/census/README.md]: What exactly differentiates Yes vs. Historical Yes?

Comment (breville) [reply dashboard/app/models/census/README.md]: Oh, Historical Yes is when we don't have any any data for the current year?

Comment (drewsamnick) [reply dashboard/app/models/census/README.md]: Yes: https://github.com/code-dot-org/code-dot-org/pull/21464/files/d6aebb35df432c9dd984e983cb52ffc2c1eece3a#diff-ff74bc0f76cb06c0a9eb730e3ff9b416R60

Comment (drewsamnick) [reply dashboard/app/models/census/README.md]: I updated the text to make this clearer.

**Pull request merged**